src/state/customizeDexes/updater.tsx

```diff
@@ -1,30 +1,37 @@
 import { useEffect, useMemo } from 'react'
 import { useDispatch } from 'react-redux'
 
-import { kyberswapDexes } from 'constants/dexes'
-import { ELASTIC_NOT_SUPPORTED } from 'constants/v2'
+import { KYBERSWAP_KS_DEXES_TO_UI_DEXES, KYBERSWAP_UI_DEXES } from 'constants/dexes'
 import { useActiveWeb3React } from 'hooks'
 import useLiquiditySources from 'hooks/useAggregatorStats'
 import { AppDispatch } from 'state/index'
+import { uniqueArray } from 'utils/array'
 
-import { updateAllDexes } from '.'
+import { Dex, updateAllDexes } from '.'
 
 export default function Updater(): null {
   const dispatch = useDispatch<AppDispatch>()
 
-  const { chainId, isEVM } = useActiveWeb3React()
+  const { chainId } = useActiveWeb3React()
   const { data: dexes } = useLiquiditySources(chainId)
 
   // filterout kyberswap dexes, will hardcode
   const normalizeDexes = useMemo(() => {
-    const dexesOutsideKyberswap =
-      dexes?.map(item => ({ ...item, id: item.dexId })).filter(item => !item.dexId.includes('kyberswap')) || []
-    const isSupportKSElastic = !ELASTIC_NOT_SUPPORTED[chainId]
-    return [
-      ...dexesOutsideKyberswap,
-      ...(isEVM ? kyberswapDexes.filter(item => isSupportKSElastic || item.id !== 'kyberswapv2') : []),
-    ]
-  }, [dexes, chainId, isEVM])
+    const dexesFormatted: Dex[] = dexes?.map(item => ({ ...item, id: item.dexId, sortId: item.id })) || []
+    const dexesOutsideKyberswap = dexesFormatted.filter(item => !item.id.includes('kyberswap'))
+    const dexesKyberswap = uniqueArray(
+      dexesFormatted.filter(dex => KYBERSWAP_KS_DEXES_TO_UI_DEXES[dex.id]),
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      dex => KYBERSWAP_KS_DEXES_TO_UI_DEXES[dex.id]!,
```

Comment:
update to
`dex => KYBERSWAP_KS_DEXES_TO_UI_DEXES[dex.id]`
I don't see any warning